.github/workflows/update-font.yml

```diff
@@ -0,0 +1,38 @@
+name: Font update check
+
+# **What it does**: Checks that the font cachebuster parameter has been updated if the font file has changed
+# **Why we have it**: To make sure that browser font caching is invalidated and the updated font is used
+# **Who does it impact**: Any PR which changes the font file
+
+on:
+  pull_request_target:
+    types: [opened, reopened]
```

Comment:
Should we also use `synchronize`, or maybe simply remove the event list, so it is triggered on all?
Otherwise it could happen, that someone e.g. pushed a font update after the PR was already opened, and the action wouldn't run again.

Reply:
Done :+1: